src/_data/checklists.json

```diff
@@ -454,101 +508,64 @@
 				"checkboxId": "avoid-flashes",
 				"wcag": "2.3.1 Three Flashes or Below Threshold",
 				"url": "https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html",
-				"description": "Certain kinds of strobing or flashing animations will trigger seizures. Others may be distracting and disruptive, especially for certain kinds of cognitive disability such as ADHD."
+				"description": "Certain kinds of strobing or flashing animations will trigger seizures. Others may be distracting and disruptive, especially for certain kinds of cognitive disability such as ADHD.",
+				"wcagLevel": "A"
 			},
 			{
 				"title": "Provide a mechanism to pause background video.",
 				"checkboxId": "pause-background-video",
 				"wcag": "2.2.2 Pause, Stop, Hide",
 				"url": "https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html",
-				"description": "Background video can be distracting, especially if content is placed over it."
+				"description": "Background video can be distracting, especially if content is placed over it.",
+				"wcagLevel": "A"
 			},
 			{
 				"title": "Make sure all animation obeys the <code>prefers-reduced-motion</code> media query.",
 				"checkboxId": "prefers-reduced-motion",
 				"wcag": "2.3.3 Animation from Interactions",
 				"url": "https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html",
-				"description": "Remove animations when the “reduce motion” setting is activated. If an animation is necessary to communicate meaning for a concept, slow its duration down."
+				"description": "Remove animations when the “reduce motion” setting is activated. If an animation is necessary to communicate meaning for a concept, slow its duration down.",
+				"wcagLevel": "AAA"
 			}
 		]
 	},
 	"color contrast": {
-		"preface": "<a class=\"c-preface__link\" href=\"{{ '/posts/2015-01-05-what-is-color-contrast/' | url }}\">Color contrast</a> is how legible colors are when placed next to, and on top of each other.",
-		"tasks": [
```

Comment:
is the removal of this tasks correct?

Reply:
Another mistake from when I was working with the diff. Fixed :)